changelog.d/20250624_130027_sekachev.bs_fixed_confirmation_email.md

```diff
@@ -0,0 +1,4 @@
+### Changed
+
+- Improved email templates for email vericiation and organization invitation
```

Comment:
Some more detail would be good to see. At least the URL fix should be a separate item (in the "Fixed" category).

Also, vericiation -> verification (or maybe confirmation).